### DISCUSSION TITLE
Is there any way to collect lineage/hot spot of data in Trino engine with gateway Kyuubi？

### DISCUSSION THREAD

**Discussion by madeirak** [apache/kyuubi#6811] [Q&A] [answered]
As show in https://kyuubi.readthedocs.io/en/v1.9.2/extensions/engines/spark/lineage.html#
column level lineage in Spark engine is satisfied

**Comment by pan3793** [comment, marked as answer]
I remember the OSS Trino has built-in lineage information in its events system, not sure table level or column level.
